packages/web-api-scan-job-manager/docker-image-config/Dockerfile

```diff
@@ -1,11 +1,24 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-FROM node:12-slim
+FROM mcr.microsoft.com/windows:1809-amd64
+
+ENV NODE_VERSION 12.20.2
+
+USER ContainerAdministrator
```

Comment:
do we need to keep admin level after setup done?

Reply:
we need the Container admin to run the poweshell script and create the ip static route inside the container, otherwise the script fails to run

Reply:
What I mean after script switch to `USER ContainerUser`

Reply:
Yeah I know, it doesn't work
I tried it in this commit https://github.com/microsoft/accessibility-insights-service/pull/1289/commits/ec5dd1ceb0fdc9511dbc933dedb743baa1b815af